src/Helper/Dsn.php

```diff
@@ -6,9 +6,9 @@
 
 final class Dsn
 {
-    private ?string $dbname;
+    private string $dbname;
```

Comment:
It is OK not to specify database name. That's useful, for example, when creating databases.

Reply:
This requires more code changes.